includes/hooks.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+// set up the default VPFO template as the starting page template for new pages
+function vpfo_set_default_template_on_new_page( $post_id, $post, $update ) {
+	// Only proceed if it's a page, it's not an update (new page), and it's in the admin area
+	if ( 'page' === $post->post_type && ! $update && is_admin() ) {
+		$default_template = 'vpfo-page.php'; // Replace with your custom template slug
+
+		// Check if the page has a template assigned; if not, set the default template
+		if ( get_post_meta( $post_id, '_wp_page_template', true ) === '' ) {
+			update_post_meta( $post_id, '_wp_page_template', $default_template );
```

Comment:
Please can we have some sanitization here before it hits the database. absint() for the post ID and perhaps esc_html() for the value.

Reply:
@richardtape Sanitization added, please review.